next/components/events/eventList.tsx

```diff
@@ -21,60 +17,47 @@ interface EventListItemProps {
 export function EventListItem({ event }: EventListItemProps) {
   const { t } = useTranslation();
   const router = useRouter();
-  const month = formatDateMonth(event.created, router.locale);
-  const day = formatDateDay(event.created, router.locale);
+  const month: string = formatDateMonth(event.field_event_date, router.locale);
+  const day = formatDateDay(event.field_event_date, router.locale);
+  const year = formatDateYear(event.field_event_date, router.locale);
+  console.log("date", month, day, year);
+
   return (
-    <div
-      
+
+    <Link
+      href={event.path.alias}
       className={classNames(
-        "bg-primary border flex flex-col relative group",
+        "flex flex-col justify-between relative w-80 h-80  border-finnishwinter dark:border-scapaflow border-2  dark:shadow-fog hover:shadow-lg hover:shadow-primary-100",
         event.sticky
-          ? "border-primary-100 bg-primary-50"
-          : "border-finnishwinter bg-white",
+          ? "border-primary-100 shadow-md shadow-primary-100 dark:shadow-fog "
+          : "border-finnishwinte dark:bg-steelgray",
       )}
     >
-      <div className=" overflow-hidden">
+      <div className="flex flex-col">
         {event.field_event_image && (
           <Image
             src={absoluteUrl(event.field_event_image.uri.url)}
-            width={600}
-            height={440}
-            className="transition-transform duration-300 ease-in-out group-hover:scale-125"
+            width={300}
+            height={300}
+            className="relative object-cover w-80 h-56"
             alt={event.field_event_image.resourceIdObjMeta.alt}
           />
         )}
-        <div className="absolute top-2 left-2">
-        <div className=" bg-secondary-600 py-3 px-5 rounded-lg font-bold">
+        <div className=" flex flex-row justify-around items-center w-28 h-8 -mt-10 ml-2 z-20 bg-rose dark:text-steelgray  font-bold">
```

Comment:
For this event date tag I would maybe add a bit more padding on the sides, maybe try px-2 if it does work with set width and height (if not ignore this).
Rounded corners would nice here like rounded or rounded-md.

Reply:
I'm a little bit doubting about rounded corners because we don't have many rounded corners.. I would even say we don't have it at all. so in that way it will be a little bit out of general styling, what do you think
But it still looks nice 
Idk
![Screenshot 2023-12-15 at 0 04 25](https://github.com/BCH-Wunder-Project-team-4/Wunder.io/assets/117649417/2c018b17-a6c4-417a-999a-007994c614bb)
?

Reply:
It looks nice but actually the extra space on the sides is probably what it needed most. I think we can go without the rounded corners in this tag :D